bundle/manifests/o2ims-hardwaremanagement.oran.openshift.io_nodepools.yaml

```diff
@@ -73,11 +77,17 @@ spec:
                       type: array
                     name:
                       type: string
+                    resourcePoolId:
+                      type: string
+                    role:
+                      type: string
                     size:
                       type: integer
                   required:
                   - hwProfile
                   - name
+                  - resourcePoolId
+                  - role
```

Comment:
Did you manually edit this file to add these? The ci-job test is flagging this as not matching the generated file

Reply:
I ran make manifests generate and committed the files. I did not enter these, maybe I missed a step..

Reply:
Updated missing changes with `make bundle`